### PR TITLE
fix(core): info-label, fix a11y issues

### PR DESCRIPTION
**Pull request by Betrozov**
## Related Issue.
Closes SAP/fundamental-ngx/#6207

## Description

Fixed info-label a11y issues

##### During Implementation
1. Visual Testing:
- [n/a] visual misalignments/updates
- [n/a] check Light/Dark/HCB/HCW themes
- [n/a] RTL/LTR - proper rendering and labeling
- [n/a] responsiveness(resize)
- [n/a] Content Density (Cozy/Compact/(Condensed))
- [n/a] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [n/a] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [n/a] Mouse vs. Keyboard support
- [n/a] Text Truncation
2. API and functional correctness
- [n/a] check for console logs (warnings, errors)
- [n/a] API boundary values
- [n/a] different combinations of components - free style
- [n/a] change the API values during testing
3. Documentation and Example validations
- [n/a] missing API documentation or it is not understandable
- [n/a] poor examples
- [n/a] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality
- [X] the commit message(s) follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [n/a] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [n/a] Run npm run build-pack-library and test in external application
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

##### PR Review
- [ ] visual misalignments/updates
- [ ] check Light/Dark/HCB/HCW themes
- [ ] RTL/LTR - proper rendering and labeling
- [ ] responsiveness(resize)
- [ ] Content Density (Cozy/Compact/(Condensed))
- [ ] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [ ] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [ ] Mouse vs. Keyboard support
- [ ] Text Truncation
2. API and functional correctness
- [ ] check for console logs (warnings, errors)
- [ ] API boundary values
- [ ] different combinations of components - free style
- [ ] change the API values during testing
3. Documentation and Example validations
- [ ] missing API documentation or it is not understandable
- [ ] poor examples
- [ ] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

**Review by N1XUS — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/info-label/examples/info-label-text-icon-example.component.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
-<div style="margin-top: 10px" *ngFor="let item of ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']">
+<div style="margin-top: 10px" *ngFor="let color of ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']">
     <fd-info-label
         type="icon"
         glyph="hide"
         label="Info label"
-        [color]="item"
+        [color]="color"
         title="Info Label with Icon"
+        [ariaLabel]="'Info Label, type - icon, color scheme - ' + color"
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/info-label/examples/info-label-text-icon-example.component.html:8]: how would i18n work here?

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Screen Shot 2021-09-16 at 11 00 07 AM

On accessing the element with the tab key we get this focus outline which is not per Fiori specs.

File: apps/docs/src/app/core/component-docs/info-label/examples/info-label-text-icon-example.component.html (hunk 1 above)
Comment (Betrozov) [reply apps/docs/src/app/core/component-docs/info-label/examples/info-label-text-icon-example.component.html:8]: it is just example
Or what do you mean?

**Comment by InnaAtanasova** [comment]
The macos default screen reader is reading `INFO LABEL` 3 times. Why do we have title property 2 times?


Screen Shot 2021-09-16 at 1 33 06 PM

**Comment by InnaAtanasova** [comment]
Info label is non-interactive element, it should not receive a focus or tab stop.
Please refer to this screenshot taken from https://accessibilityinsights.io/docs/en/windows/reference/tabstops/
Screen Shot 2021-09-16 at 3 38 34 PM

**Pull request closed without merging**